channels/af.m3u

```diff
@@ -10,9 +10,9 @@ https://d10rltuy0iweup.cloudfront.net/ATNNAT/myStream/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Pushto" tvg-logo="https://i.imgur.com/8VbuI9n.png" group-title="",Ariana TV
 https://d2g7v53450s2i2.cloudfront.net/ATNUS/streamdelay/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="https://i.imgur.com/8VbuI9n.png" group-title="",Ariana TV US
-http://d2g7v53450s2i2.cloudfront.net/ATNUS/streamdelay/chunklist.m3u8
+https://d2g7v53450s2i2.cloudfront.net/ATNUS/streamdelay/chunklist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Pushto" tvg-logo="https://i.imgur.com/XpR1VvZ.png" group-title="",Kayhan TV
-http://playout395.livestreamingcdn.com:1935/live/Stream1/playlist.m3u8
+https://playout395.livestreamingcdn.com/live/Stream1/playlist.m3u8
```

Comment:
New link does not work, original link works.

Reply:
Hi, this one works fine in VLC & via streamtest https://streamtest.in/logs/https-playout395-livestreamingcdn-com-live-stream1-playlist-m3u8-95d3da00-de0c-4e68-a3d7-e514cf53cd1c